crates/core/component/dao/src/component/view.rs

```diff
@@ -38,6 +47,17 @@ impl<T> StateReadExt for T where T: StateRead + ?Sized {}
 
 #[async_trait]
 pub trait StateWriteExt: StateWrite {
+    /// Writes the provided DAO parameters to the JMT.
+    fn put_dao_params(&mut self, params: DaoParameters) {
+        // TODO: this needs to be handled on a per-component basis or possibly removed from the compact block
+        // entirely, currently disabled, see https://github.com/penumbra-zone/penumbra/issues/3107
+        // Note to the shielded pool to include the chain parameters in the next compact block:
+        // self.object_put(state_key::chain_params_changed(), ());
```

Comment:
Can we flag this in that issue, so we don't forget? Or maybe add a checklist item to that issue to do `rg 3107` and ensure that all the references to the issue were resolved as part of the fix?

Reply:
The latter seems like less work and more resilient to forgetting to tag something on the issue.

Reply:
Yeah that was my intent with leaving the link to the issue everywhere, I'll make sure to add that to #3107